package.json

```diff
@@ -25,11 +25,12 @@
     "libs-test-all": "npm-run-all libs:test:*",
     "libs-lint-all": "npm-run-all libs:lint:*",
     "libs:watch:config": "ng build @valtimo/config --watch",
+    "libs:watch:access-control": "ng build @valtimo/access-control --watch",
```

Comment:
Shouldn't acces-control be one if the first, if not the first in the build order? I can imagine even @valtimo/config might want to make use of its functionality